Sources/ComposableArchitecture/Effects/Cancellation.swift

```diff
@@ -27,6 +27,30 @@ extension Effect {
   ///     canceled before starting this new one.
   /// - Returns: A new effect that is capable of being canceled by an identifier.
   public func cancellable(id: AnyHashable, cancelInFlight: Bool = false) -> Effect {
+    // NB: This check intends to work around bugs over different versions of Combine
+    #if swift(>=5.3) && !os(macOS)
```

Comment:
There is probably a better check to perform here but this seems to work when running Catalina through Xcode 11.6 and 12 beta 3.

Still need to confirm Big Sur beta 3 behavior...